### PR TITLE
adding dark purple v2

### PR DESCRIPTION
**Pull request by zoranikolic**

### CODE REVIEW

**Comment by dario92** [comment]
Hey @zoranikolic it looks ok to me but I am not too familiar with CORE's codebase, @danciu can you please have a look at this PR ?

File: src/sass/dist/core.scss [hunk 1]
```diff
@@ -666,6 +666,14 @@
   color: $white;
 }
 
+.bg-darkPurple2,
+.bg-dark-purple2,
+.bg-darkpurple2 {
```
Comment (danciu) [inline src/sass/dist/core.scss]: Are we using all these class names in Presales? Thanks

Comment (zoranikolic) [inline src/sass/dist/core.scss]: Hey @danciu no, only using bg-darkPurple2 for Mobile. I copied the above - i will delete the two extra classes?

File: src/sass/dist/core.scss [hunk 2]
```diff
@@ -666,6 +666,12 @@
   color: $white;
 }
 
+.bg-darkPurple2{
```
Comment (danciu) [inline src/sass/dist/core.scss]: Great, thanks @zoranikolic. Let's also add an extra space between the class name and the curly bracket please, and then it's ready to go. 👍

**Comment by zoranikolic** [comment]
:+1: @danciu

**Comment by danciu** [comment]
💯 thanks :)

**Pull request merged**